README.md

```diff
@@ -15,29 +16,36 @@ library.
 Dependencies and other Requirements
 -----------------------------------
 
-Supports (tested)  Ubuntu patched with RT-preempt. 
-Supports (experimental) Mac OS , Ubuntu patched with Xenomai
+- Supports Ubuntu patched with PREEMPT_RT patched kernel (see documentation on
+  [how to install the PREEMPT_RT
+  kernel](http://people.tuebingen.mpg.de/mpi-is-software/robotfingers/docs/robot_interfaces/doc/real_time.html)).
+- Experimental suport for Mac OS and Ubuntu patched with Xenomai
 
-(see: https://github.com/machines-in-motion/real_time_tools)
+See also the [real_time_tools](https://github.com/machines-in-motion/real_time_tools) package.
 
 A real-time CAN driver has to be loaded and the interface to be set up.  This
-can be done with the script `start_rtcan` that is included in the xenomai-core
-repository (https://git-amd.tuebingen.mpg.de/amd-clmc/xenomai-core).
+can be done with the script `start_rtcan` that is included in the [xenomai-core
+repository](https://git-amd.tuebingen.mpg.de/amd-clmc/xenomai-core).
+With PREEMPT_RT we also made good experience using the default driver, though.
```

Comment:
I'm a bit unsure about this.  I am simply using the default drivers that come with the kernel (that's why I added the last line).  I don't know if the preempt_rt patch modifies them in any way to make them more "real-time" or if things could actually be improved by using a different driver here.  Do you happen to know something about this?

Reply:
The last time I checked (~2.5 years ago), PEAK didn't had any specific drivers for RT_PREEMPT (I asked the company support). From experience, it seems the official linux drivers work well with RT_PREEMPT, no?